scripts/publish/combine_minify_js.R

```diff
@@ -8,6 +8,20 @@ publish.combine_minify_js <- function(viz) {
   
   # combine
   file_connection <- file(viz[["location"]])
+  
+  # first add vizlab resources
```

Comment:
I think this is fine for this viz, but in the future, we probably want it to loop through js resources in the order they were declared, as opposed to these being first because they are libraries. Here, we want them first anyways, but this kind of changes the order behind the scenes as opposed to giving the js library order control to the args in the viz.yaml